docs/develop/run.md

```diff
@@ -48,37 +48,163 @@ $ curl -k -XGET https://localhost:5000/api/records/ | python3 -m json.tool
     "hits": {
         "hits": [
             {
-                "created": "2019-12-19T20:15:29.167218+00:00",
-                "id": "s2pwq-mzw48",
+                "created": "2020-02-25T15:54:52.127129+00:00",
+                "updated": "2020-02-25T15:54:52.127134+00:00",
+                "revision": 0,
+                "id": "zgxnf-z7n12",
                 "links": {
-                    "files": "https://localhost:5000/api/records/s2pwq-mzw48/files",
-                    "self": "https://localhost:5000/api/records/s2pwq-mzw48"
+                "files": "https://localhost/api/records/zgxnf-z7n12/files",
+                "self": "https://localhost/api/records/zgxnf-z7n12"
                 },
                 "metadata": {
                     "_access": {
                         "files_restricted": false,
                         "metadata_restricted": false
                     },
+                    "_created_by": 2,
+                    "_default_preview": "previewer one",
+                    "_internal_notes": [
+                        {
+                        "note": "RDM record",
+                        "timestamp": "1981-12-29",
+                        "user": "inveniouser"
+                        }
+                    ],
+                    "_owners": [
+                        1
+                    ],
                     "access_right": "open",
+                    "community": {
+                        "primary": "Maincom",
+                        "secondary": [
+                        "Subcom One",
+                        "Subcom Two"
+                        ]
+                    },
+                    "contact": "info@inveniosoftware.org",
                     "contributors": [
                         {
-                            "name": "Jeanne Jones"
+                        "affiliations": [
+                            {
+                            "identifier": "entity-one",
+                            "name": "Doyle, Miller and Williams",
+                            "scheme": "entity-id-scheme"
+                            }
```

Comment:
I think something weird happened with the indentation across this file. We should re-indent appropriately.